cypress/integration/EnrollmentPage/WidgetEnrollment.feature

```diff
@@ -1,25 +1,25 @@
 Feature: User interacts with Enrollment Widget
 
-    Scenario: User can close the Enrollment Widget
-        Given you open the enrollment page
-        When you click the enrollment widget toggle open close button
-        Then the enrollment widget should be closed
+    # Scenario: User can close the Enrollment Widget
```

Comment:
Did you intend to remove all these tests? (this file and TeiWorkingListsUser.feature)